### ISSUE TITLE
Screen reader does not announce checked & unchecked state of check boxes in windows.

### ISSUE THREAD

**Issue by msftedad** [ant-design/ant-design#55944]
**Test Environment:**
OS: Windows 11 Version25H2 (OS Build 26220.7271)
Browser: Edge (Version 143.0.3650.46)
Screen Reader: NVDA (2025.3.1)
URL: https://ant.design/components/tree-select?locale=en-US

**User Impact:**
Screen reader users will be impacted because the checked and unchecked states of checkboxes are not announced. This makes it impossible for them to understand selection status, leading to errors and reduced accessibility.

**Pre-Requisites:**
Enable NVDA

**Repro Steps:**

1. Open the URL: https://ant.design/components/tree-select?locale=en-US in edge browser.
2. 'Tree Select - Ant Design' page will open.
3. Press tab key to reach to the dropdown control and expand it.
4. checkboxes will appear.
5. Verify whether screen reader does not announce checked & unchecked state of check boxes in windows or not.6. 

**Actual Result:**
Screen reader does not announce checked & unchecked state of check boxes in windows.
Screen reader remains silent

**Refer Attachment:**

1. Screen reader does not announce checked & unchecked state of check boxes in windows.png
2. Screen reader does not announce checked & unchecked state of check boxes in windows.mp4

https://github.com/user-attachments/assets/3bf3e110-d5e8-4d4b-8515-a60ad364dcb6
<img width="974" height="543" alt="Image" src="https://github.com/user-attachments/assets/b4737350-87b9-4962-bd60-8585fe14f952" />

**Expected Result:**
Screen reader should announce checked & unchecked state of check boxes in windows.

| Environment | Info |
| --- | --- |
| antd | 5.27.6 |
| React | 18 |
| System | Windows 11 Version25H2 (OS Build 26220.7271) |
| Browser | Edge (Version 143.0.3650.46) |

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by aojunhao123** [comment]
let me try to fix it

**Comment by msftedad** [comment]
Hi @aojunhao123, is there any update on this issue?
